staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -230,6 +230,7 @@ func (s *EtcdOptions) Complete(
 
 	if len(s.EncryptionProviderConfigFilepath) != 0 {
 		ctxServer := wait.ContextForChannel(stopCh)
+		// nolint:govet // The only code path where closeTransformers does not get called is when it gets stored in dynamicTransformers.
```

Comment:
cc @enj for review

Reply:
The current code is correct, but the comment should mention that dynamic transformers takes ownership of the transformers and their cancellation.

Reply:
Added.